### DISCUSSION TITLE
PCB capability with gen 1 nest

### DISCUSSION THREAD

**Discussion by aaronpowell** [justLV/onju-voice#69] [Q&A]
I've got a gen 1 nest mini that's dead so I thought I'd experiment with this project in it. Obviously there's some hardware feature difference between the gen 1 and 2, but how size compatible are the PCB's?

**Comment by Michael-Kaempf** [comment]
Same question from me.

**Comment by Hedda** [comment]
See these two indirectly related requests where some other PCB developers are discussion creating new alternative PCBs for Google Mini's:

* https://github.com/justLV/onju-voice/issues/59

* https://github.com/justLV/onju-voice/issues/74

Also see existing feature request issue and discussion that was closed by original developer for this project has no plans on doing this himself:

* https://github.com/justLV/onju-voice/issues/27

In summery; someone else needs to step up and make this and all of those happen for it to become a reality.

PS: FYI, for reference, see related requests closed as "not planed" by the original developer: 

* https://github.com/justLV/onju-voice/issues?q=is%3Aissue+is%3Aclosed+nest
  * https://github.com/justLV/onju-voice/issues/27  
  * https://github.com/justLV/onju-voice/issues/28
  * https://github.com/justLV/onju-voice/issues/29
  * https://github.com/justLV/onju-voice/issues/36
